core/src/main/java/hudson/PluginManager.java

```diff
@@ -446,6 +478,40 @@ public void run(Reactor reactor) throws Exception {
         }});
     }
 
+    @Extension
+    public final static PluginManagerAdministrativeMonitor NOTICE = new PluginManagerAdministrativeMonitor();
+
+    /**
+     * Administrative Monitor for failed plugins
+     */
+    public static final class PluginManagerAdministrativeMonitor extends AdministrativeMonitor {
+        public final List<String> pluginError = new ArrayList<>();
+
+        void addErrorMessage(String error) {
+            pluginError.add(error);
+        }
+
+        public boolean isActivated() {
+            return !pluginError.isEmpty();
+        }
+
+        /**
+         * Depending on whether the user said "dismiss" or "correct", send him to the right place.
+         */
+        public void doAct(StaplerRequest req, StaplerResponse rsp) throws IOException {
+            if(req.hasParameter("dismiss")) {
+                disable(true);
```

Comment:
Not sure whether this should even be possible (and survive a restart). What happens when new issues come up? The monitor should re-appear.